accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,475 @@
+# Marking Windows-only APIs
```

Comment:
was the addition of the 14MB platform-comapt-data.xlsx intentional? (commenting here since I can't comment on the binary file)

Reply:
@terrajobst noted during the API review meeting that this was intentional so that the analysis that went into this could be captured. The thought was that since this is the `designs` repo, it's OK to include the binaries in it.